packages/loader/container-loader/src/containerContext.ts

```diff
@@ -100,8 +100,9 @@ export class ContainerContext implements IContainerContext {
         return this.taggedLogger;
     }
 
+    // will be removed after 0.55
     public get id(): string {
```

Comment:
Given that loader is pretty much moves slower than runtime, any changes here mean latest versions runtime no longer calls it. If so, it feels it's better to remove it vs. just return empty string.
I.e. I'm not sure what we get by returning empty string, it feels like with same effect we can either remove API or just throw here.

Reply:
i just changed it to "" to test if everything will build. i will remove eventually!